capa/main.py

```diff
@@ -698,11 +702,11 @@ def install_common_args(parser, wanted=None):
 
     if "signatures" in wanted:
         parser.add_argument(
-            "--signature",
-            dest="signatures",
+            "-s",
```

Comment:
is this option used commonly enough to warrant a shortcut?

Reply:
I've added it for consistency with `--rules` which is likely used as often(?) 😕

Reply:
same for `--backend`?

Reply:
ok, fine with me